src/hooks/useDataSync/useDataSync.tsx

```diff
@@ -85,16 +85,16 @@ export const useDataSync: UseDataSync = () => {
   }
 
   const resetCaches = () => {
+    const now = Date.now()
     setSyncTimestamps({
-      [DataModel.BALANCE_SHEET]: initialTimestamp,
-      [DataModel.CHART_OF_ACCOUNTS]: initialTimestamp,
-      [DataModel.JOURNAL]: initialTimestamp,
-      [DataModel.LEDGER_ACCOUNTS]: initialTimestamp,
-      [DataModel.LINKED_ACCOUNTS]: initialTimestamp,
-      [DataModel.PROFIT_AND_LOSS]: initialTimestamp,
-      [DataModel.STATEMENT_OF_CASH_FLOWS]: initialTimestamp,
+      [DataModel.BALANCE_SHEET]: now,
+      [DataModel.CHART_OF_ACCOUNTS]: now,
+      [DataModel.JOURNAL]: now,
+      [DataModel.LEDGER_ACCOUNTS]: now,
+      [DataModel.LINKED_ACCOUNTS]: now,
+      [DataModel.PROFIT_AND_LOSS]: now,
+      [DataModel.STATEMENT_OF_CASH_FLOWS]: now,
     })
-    setReadTimestamps({})
```

Comment:
If we reset these time `undefined`, then the components think we don't have stale data _because we don't have any data_. 

We want to communicate that this data has been updated like any other data update, so we leave the read timestamps in place so the cache thinks there _is_ data, it's just out of date and needs to be refreshed.